docs/content/dagster-cloud/getting-started/getting-started-with-hybrid-deployment.mdx

```diff
@@ -102,7 +98,9 @@ In this step, you'll run your first job in Dagster Cloud.
 
 ---
 
-## Step 5: Set up Continuous Integration
+## Step 5: Set up Branch Deployments
+
+In Step 3, you most likely setup CI/CD to automatically deploy code changes in your main branch to Dagster Cloud. Dagster Cloud also supports continuous integration testing through Branch Deployments. For example, you may wish to create an isolated deployment in Dagster Cloud when a pull request is created in your Git repository.
 
 Branch Deployments allow you to quickly and collaboratively develop your Dagster jobs. When a branch is created or updated in your Dagster repository, Branch Deployments will create a testing environment in the current Dagster Cloud deployment. For example, `prod`.
```

Comment:
Today this is not true, branch deployments are only created for PRs by our existing action. But it could be true if they modified the action themselves, so I left the language as-is.